### DISCUSSION TITLE
More output

### DISCUSSION THREAD

**Discussion by ghost** [jensrossbach/node-red-contrib-chronos#53] [Q&A] [answered]
Hi,
if I need to set two or more output by the same single input is correct to repeat the same input for all the output?

![chrono](https://user-images.githubusercontent.com/81916825/127633461-3cb93a46-f560-4fff-9b80-adf237389f99.png)

**Comment by jensrossbach** [comment, marked as answer]
Hi,

I'm not sure if I understand correctly what you want to achieve. The scheduler node doesn't really have inputs (except the control inputs), so what exactly is your use case? Do you want to have multiple outputs being scheduled at the same time?

**Reply by ghost** [reply]
> Do you want to have multiple outputs being scheduled at the same time?

Yes......; my example is correct?

**Reply by jensrossbach** [reply]
I never tried exactly this scenario on my own, but I don't see a reason why it shouldn't work. So yes, this looks ok.

**Reply by ghost** [reply]
OK, thank you ...

**Reply by jensrossbach** [reply]
I just had some more thoughts about your use case and I think there is an alternative way to achieve the same: you can add a single schedule event and set the output type to "full message". Then you define an array of messages in JSON which results in multiple messages being sent on the same schedule event. Of course this only works if you don't need the two output messages at different output ports.

This could look something like this:
![image](https://user-images.githubusercontent.com/5118924/127779435-b23a51ca-ce7f-41b9-ba0a-1360d654a62c.png)
